File: studio/src/main/java/br/com/api/enums/TypeOfSalaryEnum.java
package br.com.api.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum TypeOfSalaryEnum {

	BY_HOUR("label.by.hour"), 
	BY_PERCENT("label.by.percent"), 
	BY_MONTH("label.by.month");

	private final String i18n;
}

File: src/model/TypeOfSalaryEnum.ts
export enum TypeOfSalaryEnum {
    BY_HOUR = 'Por hora',
    BY_PERCENT = 'Por percentual',
    BY_MONTH = 'Por mês'
}

export const TypeOfSalaryList: {
    key: string;
    value: string;
}[] = Object.entries(TypeOfSalaryEnum).map(([key, value]) => ({ key, value }))
